core/src/org/samtools/htsjdk/core/Locatable.java

```diff
@@ -0,0 +1,140 @@
+package org.samtools.htsjdk.core;
+
+import java.util.OptionalLong;
+
+/**
+ * Represents the location of a sequence with respect to another sequence (reference).
+ *
+ * <p>
+ * This interface represents a location or "mapping" with respect to a reference sequence. Examples
+ * of this abstraction are features located on the genome or alignment between two sequences (query
+ * with respect to a reference).
+ * </p>
+ *
+ * @author Daniel Gomez-Sanchez (magicDGS)
+ * @implSpec note that 0-length intervals are allowed in an implementation-dependent manner.
+ */
+public interface Locatable {

```

Comment:
Maybe we should extend `Comparable`, but I think that it will be better to have a `LocatableComparator` instead to allow different comprators to be use

Reply:
I don't think this should be comparable because there's no single consistent comparator, it depends on how you want to sort he chromosomes.

Reply:
Won't do.